server/src/main/java/org/opensearch/node/remotestore/RemoteStoreNodeService.java

```diff
@@ -42,31 +42,21 @@ public class RemoteStoreNodeService {
     private final Supplier<RepositoriesService> repositoriesService;
     private final ThreadPool threadPool;
     public static final Setting<CompatibilityMode> REMOTE_STORE_COMPATIBILITY_MODE_SETTING = new Setting<>(
-        "remote_store.compatibility_mode",
+        "cluster.remote_store.compatibility_mode",
```

Comment:
This would require a documentation update. Can we live with the existing setting name for now?

Reply:
Raised a documentation [PR](https://github.com/opensearch-project/documentation-website/pull/7342) already . We should go GA with correct  setting name only . Else it will be confusing for the users later on .

Reply:
Let's add that to the changelog also in that case